### DISCUSSION TITLE
第13期：OpenAI 可能在 12 个月内破产，预计亏损 50 亿美元 | 信息差——独立开发者出海周刊

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/gapis.money#17] [General]
# 第13期：OpenAI 可能在 12 个月内破产，预计亏损 50 亿美元 | 信息差——独立开发者出海周刊

Knowledge is power, info-gap is money!「信息差——独立开发者出海周刊」是一个帮助独立开发者缩小信息差的技术周刊。

https://gapis.money/weekly/2024-08-05_013

**Comment by flashlab** [comment]
好多干货
